eng/pipelines/templates/variables/image.yml

```diff
@@ -2,27 +2,17 @@
 
 variables:
   - name: LINUXPOOL
-    value: azsdk-pool-mms-ubuntu-2004-general
-  - name: LINUXNEXTPOOL
-    value: azsdk-pool-mms-ubuntu-2204-general
+    value: azsdk-pool
   - name: WINDOWSPOOL
-    value: azsdk-pool-mms-win-2022-general
+    value: azsdk-pool
   - name: MACPOOL
     value: Azure Pipelines
 
   - name: LINUXVMIMAGE
-    value: azsdk-pool-mms-ubuntu-2004-1espt
+    value: ubuntu-22.04
   - name: LINUXNEXTVMIMAGE
-    value: azsdk-pool-mms-ubuntu-2204-1espt
+    value: ubuntu-22.04
```

Comment:
can we remove `LINUXNEXTVMIMAGE` also?

Reply:
No, because chances are, we'll have a new NEXT image soon.

Reply:
@billwert, the LINUXNEXTIMAGE will be ubuntu-24.04 but this is blocked until it's supported by our pool provided. [This is the issue we have tracking that.](https://github.com/Azure/azure-sdk-tools/issues/9737)